demo/src/index.js

```diff
@@ -11,11 +11,5 @@ import './lists/lists-page';
 import router from './core/router';
 
 // Initialize the router with the current path or 'examples' if none is found
-router.initBase();
 router.fallback = 'examples';
-
-const url = new URL(window.location.href);
-const path = url.pathname;
-const queryParams = Object.fromEntries(url.searchParams.entries());
-
-router.handleRouteChange(path, queryParams);
+router.start();
```

Comment:
@jboix why not keep`fallback` property private and use an option to initialize the `router`?

Something like:
```javascript
new router({
  defaultPath: 'examples', // or another self-described property
  // ... other properties
});
```

Or 

```javascript
router.init({
  defaultPath: 'examples',
  // ... other properties
});
```

Reply:
You are right it makes more sense, I've added it to the `start` function.